docs/bsp.md

```diff
@@ -739,23 +858,19 @@ trait CompileResult {
 }
 ```
 
-where `StatusCode` is defined as follows
+Notifications:
+
+The beginning of a compilation unit may be signalled to the client with a `build/taskStart` notification.
+For compilation of a build target, the notification params should include `CompileTask` object:
```

Comment:
Same question as for `TestTask`, can you clarify "For compilation of a build target, the notification params should include `CompileTask` object:"?